islandora_gsearcher.drush.inc

```diff
@@ -4,12 +4,13 @@
  * Drush integration
  */
 
+require_once "includes/utilities.inc";
```

Comment:
Could get into trouble with relative paths in some rare cases with some versions of PHP or multisite magic or permissions BS ... we usually tack `dirname(__FILE__)` onto the start of `require_once`s (i.e., 
```php
require_once dirname(__FILE__) . '/includes/utilities.inc';
```
); should be a bit safer.